Design/Rule0023AlwaysSpecifyFieldgroups.cs

```diff
@@ -40,5 +45,11 @@ private void CheckFieldgroups(SymbolAnalysisContext ctx)
                 ctx.ReportDiagnostic(Diagnostic.Create(DiagnosticDescriptors.Rule0000ErrorInRule, ctx.Symbol.GetLocation(), new Object[] { "Rule0023", "ArgumentOutOfRangeException", "" }));
             }
         }
+
+        private static bool IsTableOfTypeSetupTable(IFieldSymbol field)
+        {
+            // The first field of the table should be of type Code and exactly (case sensitive) called 'Primary Key'
+            return (field.GetTypeSymbol().NavTypeKind == NavTypeKind.Code && field.Name == "Primary Key");
```

Comment:
... so you can also check if the table only has a single field as PK, just to be sure